app/Notifications/UserEmailNotification.php

```diff
@@ -24,9 +24,9 @@ abstract class UserEmailNotification extends EmailNotification
     /**
      * Default constructor.
      *
-     * @param PublicAdministration $publicAdministration the public administration
+     * @param $publicAdministration the public administration
      */
-    public function __construct(?PublicAdministration $publicAdministration = null)
+    public function __construct($publicAdministration = null)
```

Comment:
Come mai rimuovi questo type check? Se non ci sono motivi particolare, ritengo che sia meglio lasciarlo e prevedere una variabile in più per le PA "purged" invece che allentare un controllo su tutte le notification che ereditano da questa classe per risparmiare una variabile.

Reply:
Se per la PA _purged_ passo anche l'indirizzo email da usare non ho neanche bisogno di modificare questa classe.